packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.html

```diff
@@ -0,0 +1,111 @@
+<h1 id="dedicatedCloud_datacenter_backup_header" data-translate="dedicatedCloud_datacenter_backup_title"></h1>
+<div data-ui-view>
+    <div class="datacenter-backup">
+        <div data-ovh-alert="{{ ::$ctrl.alertMain }}"></div>
+        <oui-switch data-model="true" data-on-change="$ctrl.deactivate()">
+        </oui-switch>
+        <span data-translate="dedicatedCloud_datacenter_backup_enabled"></span>
+
+        <section class="datacenter-backup-offers">
+            <h4
```

Comment:
No `h1`, no `h2` and not `h3` above

Reply:
There is a `h1` at the top of the page. `h2` or `h3` makes the text bigger than the required size

Reply:
Maybe a class should be used if this is for style purpose

Reply:
The class has been added. Thanks for the suggestion :)